dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/Microsoft.TeamsAI.csproj

```diff
@@ -1,10 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-	<TargetFramework>netstandard2.0</TargetFramework>
-	<LangVersion>latest</LangVersion>
-	<ImplicitUsings>enable</ImplicitUsings>
+	  <TargetFramework>netstandard2.0</TargetFramework>
+	  <LangVersion>latest</LangVersion>
+	  <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+    <PackageId>Microsoft.TeamsAI</PackageId>
+    <Version>1.0.0</Version>
```

Comment:
Shall we use certain beta or preview version?

Reply:
that makes sense to me, do we have a convention for versioning we follow like semantic? cc @corinagum

Reply:
I think it makes sense to use `alpha` for private preview then `beta` for public preview